index.html

```diff
@@ -23,7 +23,9 @@
 <body>
   <main>
     <div class="container header">
-      <h2> Code Journal </h2>
+      <div class="row column-full">
```

Comment:
No one element should be both a row and a column. You should give this row a child of a column, and make the `h2` a child of that column.